api/controllers/AuthController.js

```diff
@@ -68,12 +68,8 @@ module.exports = {
    * @param {Object} res
    */
   logout: function (req, res) {
-    req.logout();
-
-    // mark the user as logged out for auth purposes
-    req.session.authenticated = false;
-
-    res.redirect('/');
+    req.session.destroy();
```

Comment:
What's the difference here with what it used to do? Is this purely because it's a POST request now?